machine_management/creating-infrastructure-machinesets.adoc

```diff
@@ -14,49 +14,17 @@ Unlike earlier versions of {product-title}, you cannot move the infrastructure c
 
 include::modules/infrastructure-components.adoc[leveloffset=+1]
 
-[id="creating-infrastructure-machinesets-production"]
-== Creating infrastructure machine sets for production environments
+[id="creating-infra-machines-with-label_{context}"]
+== Creating machines to run infrastructure workloads
 
-In a production deployment, deploy at least three machine sets to hold infrastructure components. Both the logging aggregation solution and the service mesh deploy Elasticsearch, and Elasticsearch requires three instances that are installed on different nodes. For high availability, deploy these nodes to different availability zones. Since you need different machine sets for each availability zone, create at least three machine sets.
+You cannot create infrastructure machines directly using a machine set; you must first provision the node as a compute machine, and then configure it to run infrastructure workloads.
```

Comment:
I agree this probably deserves an explanation. I would probably prefer linking out to a KCS instead of getting into the weeds in the docs on Support decisions. Let me know if there's a resource to link to providing an explanation on this decision.

Reply:
Agreed.  Customers may still want to define different compute profiles, and MachineSets are the way to do that. If the customer wants a different compute profile for their infra, they can just follow the normal docs for doing so, then jump here for converting them to infra nodes. A KCS could handle that.

Reply:
@yuqi-zhang is this a gap in this solution? I can 100% see a customer wanting to allocate resources differently for these nodes that what the cluster has/uses (simply for performance implications/scaling, etc).